projection.tex

```diff
@@ -210,10 +210,10 @@ \subsection{The discretized three steps of projection algorithm}
 where
 %
 \begin{gather}
-\mathbf{C}(u^n_{i,j})=u^n_{i,j} \cdot \frac{u^n_{e}-u^n_{w}}{\Delta x}+v^n_{p} 
-\cdot \frac{u^n_{n}-u^n_{s}}{\Delta y} \\
-\mathbf{D}(u^n_{i,j})=\nu (\frac{u^n_{e}+ u^n_{w}-2u^n_{i,j}}{(\Delta x/2)^2}
-+\frac{u^n_{n}+ u^n_{s}-2u^n_{i,j}}{(\Delta y/2)^2})
+\mathbf{C}(v_1^n_{i,j})=v_1^n_{i,j} \cdot \frac{v_1^n_{e}-v_1^n_{w}}{\Delta x_1}+v_2^n_{p} 
```

Comment:
I think `v_1^n_{i,j}` is an invalid tex expression.  It should be like ${(v_1)}^n_{i,j}$ or something like that.